api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -54,11 +54,31 @@ message UpstreamTlsContext {
   google.protobuf.UInt32Value max_session_keys = 4;
 }
 
-// [#next-free-field: 8]
+// [#next-free-field: 9]
 message DownstreamTlsContext {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.auth.DownstreamTlsContext";
 
+  enum OcspStaplePolicy {
```

Comment:
Is Ocsp stapling of the client cert in mTLS valid? (A quick search didn't give me an answer). If it is, should this be moved out of the Downstream context to a message shared between downstream and upstream?

Reply:
Great question and in theory I'm not sure, but it doesn't seem done in practice from what I've read. I do know that the BoringSSL API for ocsp is pretty strict and you can only set responses on the server side and extract them on the client side.

Reply:
I'm fine either way on this. It sounds like it is possible per spec, but boringssl doesn't support it (and I've never seen someone use this feature).

Reply:
I'm going to leave it as-is.